README.md

```diff
@@ -31,5 +40,30 @@ too; replace `-full` with `-oss` in any of the above commands to install the
 OSS distribution. Note that the default distribution and OSS distribution of
 a product can not be installed at the same time.
 
+## How do I ensure my configuration is up to date?
+
+Run the following command to update your configuration:
+
+    brew tap --repair
+    brew update -v
+
+Verify your configuration is based on the `main` branch with:
+
+    git -C /opt/homebrew/Library/Taps/elastic/homebrew-tap status
+
+You should have the following output:
+
+    On branch main
+    Your branch is up to date with 'origin/main'.
+
 ## Documentation
 `brew help`, `man brew` or check [Homebrew's documentation](https://github.com/Homebrew/brew/blob/master/docs/README.md).
+
+## Troubleshooting
+### Calling bottle :unneeded is deprecated!
+
+When I execute `brew update`, the following warning appears
+  ```
+  Warning: Calling bottle :unneeded is deprecated! There is no replacement.
+  ```
+  This is related to your configuration not being up-to-date. Your are still using the legacy `master` branch which is not updated anymore. Please [follow the instructions to update](#how-do-i-ensure-my-configuration-is-up-to-date) your local configuration to use the `main` branch.
```

Comment:
Small tweak: use the indentation style here to be consistent with the rest of the file:
```suggestion

    Warning: Calling bottle :unneeded is deprecated! There is no replacement.

This is related to your configuration not being up-to-date. Your are still using the legacy `master` branch which is not updated anymore. Please [follow the instructions to update](#how-do-i-ensure-my-configuration-is-up-to-date) your local configuration to use the `main` branch.
```